### PR TITLE
DMS Coordinates output - add one decimal

### LINKED ISSUE #710: Coordinates DMS add one decimal

### PR DESCRIPTION
**Pull request by petzlux**
fix #710

### CODE REVIEW

**Comment by petzlux** [comment]
review please

File: geoportailv3/static/js/locationinfo/locationinfo.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="container-fluid" ng-show="ctrl.appSelector == 'locationinfo'">
   <div class="row">
-    <div class="col-xs-8">
+    <div class="col-xs-8" style="padding-left: 0px;">
```
Comment (rmichaelis) [inline geoportailv3/static/js/locationinfo/locationinfo.html]: You should maybe use a class instead of a style

**Comment by rmichaelis** [comment]
I think this is no more correct : 
https://github.com/Geoportail-Luxembourg/geoportailv3/blob/petzlux_dms/geoportailv3/static/js/coordinatestringservice.js#L53

You have to replace ```var hdms = ol.coordinate.toStringHDMS(coordinate);``` with something like that ```var hdms = toStringHDMS_(coordinate);```

If not the decimal will never be displayed.

**Comment by petzlux** [comment]
Ups ! I did a rebase and probably messed that change up there ...

File: geoportailv3/static/js/locationinfo/locationinfo.html (hunk 1 above)
Comment (petzlux) [inline geoportailv3/static/js/locationinfo/locationinfo.html]: okay!

**Pull request merged**